components/script/dom/window.rs

```diff
@@ -235,6 +236,17 @@ pub(crate) struct OngoingNavigation(u32);
 
 type PendingImageRasterizationKey = (PendingImageId, DeviceIntSize);
 
+/// An ancillary data of outstanding image request that was initiated by layout during
```

Comment:
I'm not sure what you mean with "outstanding".
And "data" in uncountable so I would drop "An".

Reply:
Yes that's seems clearer, thanks.